<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnplacedComponentsDock</class>
 <widget class="QDockWidget" name="UnplacedComponentsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unplaced Components</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="lstUnplacedComponents"/>
    </item>
    <item>
     <widget class="QComboBox" name="cbxSelectedComponent"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Footprint Preview</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add similar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddAll">
        <property name="text">
         <string>Auto-Add All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
